with-firebase-saml-login/app.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Delete app.json unless it's required for the app.

Reply:
missing scheme throws an expo-linking error